tests/foreman/longrun/test_inc_updates.py

```diff
@@ -207,28 +213,25 @@ def test_positive_noapply_api(
     versions = sorted(module_cv.read().version, key=lambda ver: ver.id)
     cvv = versions[-1].read()
     cvv.promote(data={'environment_ids': dev_lce.id})
-    # Read CV to pick up LCE ID and next_version
-    module_cv = module_cv.read()
 
-    # Get the content view versions and use the recent one. API always
-    # returns the versions in ascending order (last in the list is most recent)
-    cv_versions = module_cv.version
     # Get the applicable errata
     errata_list = get_applicable_errata(custom_repo)
     assert len(errata_list) > 0
 
     # Apply incremental update using the first applicable errata
-    entities.ContentViewVersion().incremental_update(
+    outval = entities.ContentViewVersion().incremental_update(
         data={
             'content_view_version_environments': [
                 {
-                    'content_view_version_id': cv_versions[-1].id,
+                    'content_view_version_id': cvv.id,
                     'environment_ids': [dev_lce.id],
                 }
             ],
             'add_content': {'errata_ids': [errata_list[0].id]},
         }
     )
-    # Re-read the content view to get the latest versions
-    module_cv = module_cv.read()
-    assert len(module_cv.version) > len(cv_versions)
```

Comment:
Can you explain why this assertion is not good and is being removed?

Reply:
Yea, there was reported BZ for unsorted cv version, In this test module we are ensuring api increamental update of applicable errata working fine without error, we are able to assert this change.

Reply:
Hope so I have answered your query.